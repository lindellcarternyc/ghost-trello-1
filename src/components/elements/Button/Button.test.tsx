import * as React from 'react'
import * as ReactDOM from 'react-dom'
import { Button } from './button'

it('renders a button without crashing', () => {
  const div = document.createElement('div')
  ReactDOM.render(<Button />, div)
})

it('accepts a content prop (string) and renders it', () => {
  const div = document.createElement('div')
  const button = <Button content='Button' />
  expect(button.props.content).toBe('Button')
  
})

// it('always renders')