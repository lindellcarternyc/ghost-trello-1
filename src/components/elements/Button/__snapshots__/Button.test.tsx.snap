// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`accepts a content prop (string) and renders it 1`] = `
<button>
  BUTTON
</button>
`;
